pkg/api/norman/customization/cluster/actionhandler.go

```diff
@@ -5,66 +5,22 @@ import (
 
 	"github.com/rancher/norman/httperror"
 	"github.com/rancher/norman/types"
-	gaccess "github.com/rancher/rancher/pkg/api/norman/customization/globalnamespaceaccess"
 	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/auth/requests"
 	"github.com/rancher/rancher/pkg/auth/tokens"
 	mgmtclient "github.com/rancher/rancher/pkg/client/generated/management/v3"
 	"github.com/rancher/rancher/pkg/clustermanager"
 	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/user"
-	v1 "k8s.io/client-go/kubernetes/typed/authorization/v1"
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 )
 
 // ActionHandler used for performing various cluster actions.
 type ActionHandler struct {
-	NodepoolGetter                v3.NodePoolsGetter
-	NodeLister                    v3.NodeLister
```

Comment:
Don't need NodeLister either

Reply:
since nodelister is being used in actions_kubeconfig.go file so can't remove it.